### PR TITLE
expand 403 section with lockout statement

### PR DESCRIPTION
**Pull request by rnewson**

### CODE REVIEW

**Review by janl — APPROVED** [review]

File: src/docs/src/api/basics.rst [hunk 1]
```diff
@@ -543,7 +543,12 @@ specific request types are provided in the corresponding API call reference.
 
 - ``403 - Forbidden``
 
-  The requested item or operation is forbidden.
+  The requested item or operation is forbidden. This might be because;
+
+  * your user name or roles do not match the security object of the database
+  * the request request administrator privileges but you don't have them
```
Comment (nickva) [inline src/docs/src/api/basics.rst:549]: “request” is repeated

Comment (rnewson) [reply src/docs/src/api/basics.rst]: fixed

**Review by nickva — COMMENTED** [review]
Looks better but  a tiny nitpick I think it might look better to capitalize each point. In most cases when we have a bullet points we seem to do that in the docs (unless it's a code literal of course).

Screenshot 2024-10-24 at 10 48 56 AM

**Review by nickva — APPROVED** [review]

**Review by kingkong2024 — COMMENTED** [review]
[Uploading 2010gcar.pdf…]()

**Comment by kingkong2024** [comment]
https://github.com/apache/couchdb/pull/5320#issue-2608645762
92080467_10157408383558721_1148221358686076928_n

**Pull request merged**